metrics/internal/collectors/ceph-block-pool.go

```diff
@@ -107,10 +109,21 @@ func getAllBlockPools(lister cephv1listers.CephBlockPoolLister, namespaces []str
 func (c *CephBlockPoolCollector) collectMirroringImageHealth(cephBlockPools []*cephv1.CephBlockPool, ch chan<- prometheus.Metric) {
 	for _, cephBlockPool := range cephBlockPools {
 		var imageHealth string
+
+		if !cephBlockPool.Spec.Mirroring.Enabled {
+			return
+		}
+
 		mirroringStatus := cephBlockPool.Status.MirroringStatus
-		if mirroringStatus != nil {
-			imageHealth = mirroringStatus.Summary.ImageHealth
+		if mirroringStatus == nil || mirroringStatus.Summary == nil {
+			return
```

Comment:
if mirroring is enabled, wouldn't we need to show an error when mirroring status is nil?

Reply:
I wanted to be optimistic that operator will update it when needed. But, adding an error log here doesn't hurt.
I'll fix it.